mypy/waiter.py

```diff
@@ -37,13 +37,11 @@ def __init__(self, name: str, args: List[str], *, cwd: str = None,
         self.passthrough = passthrough
 
     def start(self) -> None:
-        if self.passthrough is None:
+        if self.passthrough is None or self.passthrough < 0:
             self.outfile = tempfile.TemporaryFile()
         else:
             if self.passthrough >= 0:
```

Comment:
Oh right. Also, currently with `python runtests.py -q` I completely suppress pytest output (since that was the original behavior of `python runtests.py -q`). Should I perhaps still let it pass through its output? With `-q`, pytest output is just a bunch of dots, so it actually fits well with runtests.py output.